test/contracts.py

```diff
@@ -21,17 +20,69 @@ class BridgeContractConfig:
     next_swap_id: int
 
 DefaultBridgeContractConfig = BridgeContractConfig(
-    cap = "250000000000000000000000000",
-    reverse_aggregated_allowance = "3000000000000000000000000",
-    reverse_aggregated_allowance_approver_cap = "3000000000000000000000000",
-    lower_swap_limit = "1",
-    upper_swap_limit = "1000000000000000000000000",
-    swap_fee = "0",
-    paused_since_block = 18446744073709551615,
-    denom = "atestfet",
-    next_swap_id = 0
+    cap="250000000000000000000000000",
+    reverse_aggregated_allowance="3000000000000000000000000",
+    reverse_aggregated_allowance_approver_cap="3000000000000000000000000",
+    lower_swap_limit="1",
+    upper_swap_limit="1000000000000000000000000",
+    swap_fee="0",
+    paused_since_block=18446744073709551615,
+    denom="atestfet",
+    next_swap_id=0
 )
 
+
+@dataclass_json
+@dataclass
+class CW20BalanceObject:
+    amount: int
+    address: str
+
+
+@dataclass_json
+@dataclass
+class CW20ContractConfig:
+    name: str
+    symbol: str
+    decimals: int
+    initial_balances: list[dict[str, str]]
+
+
+DefaultCW20ContractConfig = CW20ContractConfig(
+    name="test coin",
+    symbol="TEST",
+    decimals=6,
+    initial_balances=[{
+            "amount": "3000000",
+            "address": "fetch1wurz7uwmvchhc8x0yztc7220hxs9jxdjdsrqmn"  # validator address
```

Comment:
no big deal, but not sure I like the hardcoded address here. Probably worth not having a DefaultCWContractConfig at all, it's not that bad to create in place where needed (and so use directly the addresses from `self.validator_address` or `self.delegator_address`

Reply:
Yeah that's fair! I'll give that a redo